src/main/java/us/kbase/auth2/providers/OrcIDIdentityProviderFactory.java

```diff
@@ -228,6 +233,58 @@ private OrcIDAccessTokenResponse(
 				this.accessToken = accessToken.trim();
 				this.fullName = fullName == null ? null : fullName.trim();
 				this.orcID = orcID.trim();
+				this.idToken = idToken == null ? null : idToken.trim();
+				this.mfaAuthenticated = parseAmrClaim(this.idToken);
```

Comment:
This class is a value class and shouldn't do anything else. As such, I would move the parsing method outside of this class and just pass in the mfa boolean. Also idToken is never used so it seems that could be deleted.

Reply:
Doing anything other than just saving the constructor arguments to instance variables. Think of it like a python named tuple

Reply:
:+1: